include/seqan3/range/views/minimiser.hpp

```diff
@@ -440,25 +554,24 @@ namespace seqan3::views
  * \{
  */
 
-/*!\brief                           Computes minimisers for a range of comparable values. A minimiser is the smallest
- *                                  value in a window.
- * \tparam urng_t                   The type of the range being processed. See below for requirements. [template
- *                                  parameter is omitted in pipe notation]
- * \param[in] urange                The range being processed. [parameter is omitted in pipe notation]
- * \param[in] window_values_size    The number of values in one window.
- * \returns                         A range of std::totally_ordered where each value is the minimal value for one window.
- *                                  See below for the properties of the returned range.
+/*!\brief Computes minimisers for a range of comparable values. A minimiser is the smallest value in a window.
+ * \tparam urng1_t The type of the first range being processed. See below for requirements. [template
```

Comment:
```suggestion
 * \tparam urng_t The type of the first range being processed. See below for requirements. [template
```

Reply:
Please revert all references to urange1 etc. The user docu does not know anything about a second range.